src/api/getRouteMC.ts

```diff
@@ -0,0 +1,89 @@
+import { ConfigEnv } from "./env.config";
+import { ItineraryPoint } from ".";
+
+//example url https://api.mapy.cz/v1/routing/route?apikey=eyJpIjoyNTcsImMiOjE2Njc0ODU2MjN9.c_UlvdpHGTI_Jb-TNMYlDYuIkCLJaUpi911RdlwPsAY&start=14.40094&start=50.0711&end=16.5990122&end=49.1718222&routeType=bike_road&lang=pl&format=geojson&avoidToll=false&waypoints=15.5903861%2C%2049.3967233&waypoints=16.4187328%2C%2049.3414642
+
+
+// Planning Types
+// Planning supports various types of planning, according to which it optimizes the proposed route:
+// car – fast – searches for a fast car route
+// car – fast with traffic – searches for a fast car route considering the current traffic conditions, congestion, etc. (only in the Czech Republic, without considering traffic outside the Czech Republic)
+// car – short – searches for a short car route
+// foot – fast – searches for a fast walking route
+// bicycle – road – searches for a route for road bicycles, with asphalt surface, preferring cycle paths and bike trails
+// bicycle – mountain – searches for a route for mountain bikes, preferring cycle paths and bike trails regardless of the surface
+
+export enum routeType {
+    carFast = "car_fast",
+    carFastWithTraffic = "car_fast_traffic",
+    carShort = "car_short",
+    footFast = "foot_fast",
+    bikeRoad = "bike_road",
+    bikeMountain = "bike_mountain"
+}
+
+export enum langType {
```

Comment:
Pytanie czy to nam potrzebne bo robimy i tak apkę po polsku

Reply:
jest taka opcja w endpoincie. Więc ją udostępniam